Datadog.Trace.Minimal.slnf

```diff
@@ -2,6 +2,7 @@
   "solution": {
     "path": "Datadog.Trace.sln",
     "projects": [
+      "tracer\\build\\_build\\_build.csproj",
```

Comment:
Is that on purpose? The goal of the solution filter is to include as few projects as possible

Reply:
> Is that on purpose? The goal of the solution filter is to include as few projects as possible

Ah, no, I didn't mean to commit that, I'll remove it. It's damn useful when you're working on the build like this though...

Reply:
```suggestion
```